vic/drivers/classic/src/read_soilparam.c

```diff
@@ -692,6 +696,108 @@ read_soilparam(FILE *soilparam,
             temp.Ws = temp.Ws / temp.max_moist[layer];
         }
 
+        // Soil thermal node thicknesses and positions
```

Comment:
This section was originally in the old initialize_model_state() function, but it really should go here, as node spacing is completely determined from soil parameters and global param options.  This way, all soil_con values are populated by the time we exit read_soilparam().